### PR TITLE
feat(ux): flatten wash start

### PR DESCRIPTION
**Pull request by vados-cosmonic**
#### :warning: This PR should be merged *after* #569 

Flatten `wash ctl start` into `wash start`

See:
- https://github.com/wasmCloud/wash/issues/538
- https://github.com/wasmCloud/wash/pull/569

## Related Issues

https://github.com/wasmCloud/wash/pull/569

## Release Information

`next`

## Testing

Built on platform(s)
- [x] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

Tested on platform(s)
- [x] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

### Acceptance or Integration

Added `integration_start.rs` which has tests

### CODE REVIEW

File: src/start.rs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+use crate::{
+    common::start_cmd::{
+        handle_command as handle_start_command, StartActorCommand, StartCommand,
+        StartProviderCommand,
+    },
+    OutputKind,
+};
+
+#[derive(Debug, Clone, Subcommand)]
+pub(crate) enum StartCliCommand {
```
Comment (connorsmith256) [inline src/start.rs]: What is this for? It looks like it's simply proxying to the `start_cmd` types?

(Also it looks like the clap names have bad copy-paste values)

**Review by connorsmith256 — CHANGES_REQUESTED** [review]

File: src/start.rs (hunk 1 above)
Comment (vados-cosmonic) [reply src/start.rs]: good catch, it *was* unnecessary -- thanks for the review!

**Comment by vados-cosmonic** [comment]
Note one extra concern, need to remove [the unused timeout mechanics](https://github.com/wasmCloud/wash/pull/569#discussion_r1204431114)

- [x] remove unused timeout mechanics

**Review by connorsmith256 — DISMISSED** [review]
I didn't look too closely at the "new" code, since I assume it was all moved from existing files. LGTM, and thanks for the new tests!

**Comment by vados-cosmonic** [comment]
Hey @connorsmith256 thanks for the quick review -- one thing I put in after was the removal unused timeout stuff ([mentioned here](https://github.com/wasmCloud/wash/pull/573#issuecomment-1561577362)).

**Review by connorsmith256 — APPROVED** [review]

**Review by thomastaylor312 — APPROVED** [review]

**Pull request merged**